abstractions/php/src/ApiClientBuilder.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace Microsoft\Kiota\Abstractions;
+
+use Microsoft\Kiota\Abstractions\Serialization\ParseNodeFactory;
+use Microsoft\Kiota\Abstractions\Serialization\ParseNodeFactoryRegistry;
+use Microsoft\Kiota\Abstractions\Serialization\SerializationWriterFactory;
+use Microsoft\Kiota\Abstractions\Serialization\SerializationWriterFactoryRegistry;
+use Microsoft\Kiota\Abstractions\Store\BackingStoreParseNodeFactory;
+use Microsoft\Kiota\Abstractions\Store\BackingStoreSerializationWriterProxyFactory;
+
+class ApiClientBuilder {
+    private function __construct(){}
+
+    /**
+     * Registers the default serializer to the registry.
+     * @param SerializationWriterFactory $factoryClass the class of the factory to be registered.
+     */
+    public static function registerDefaultSerializer(SerializationWriterFactory $factoryClass): void {
+        /** @var SerializationWriterFactory $factory */
+        $factory =  new (get_class($factoryClass))();
```

Comment:
From other implementations, looks like we should be accepting a `string` here & validating if the `class_exists` before instantiating

Reply:
Same for `registerDefaultSerializer()`